x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -813,4 +823,17 @@ export class AlertsClient<
   public isUsingDataStreams(): boolean {
     return this._isUsingDataStreams;
   }
+
+  private throwIfHasClusterBlockException(response: BulkResponse) {
+    response.items.forEach((item) => {
+      const op = item.create || item.index || item.update || item.delete;
+      if (op?.error && op.error.type === CLUSTER_BLOCK_EXCEPTION) {
+        throw new ErrorWithType({
+          message: op!.error!.reason,
```

Comment:
nit: safer to do op?.error?.reason ?? `fallback reason`

Reply:
done